Editor/UI/Windows/TrackingSettingsWindow.cs

```diff
@@ -30,27 +30,47 @@ private static void ShowWizard()
             }
 
             window.Show();
-            window.minSize = new Vector2(280f, 30f);
-            window.maxSize = new Vector2(280f, 30f);
+            window.minSize = new Vector2(280f, 50f);
+            window.maxSize = new Vector2(280f, 50f);
             window.Focus();
         }
 
 
         private void OnGUI()
         {
-            titleContent = new GUIContent("Analytic Settings");
+            titleContent = new GUIContent("Analytics Settings");
 
             GUILayout.Space(8);
 
             // Unknown should not be shown, so we have to remove it.
             int state = (int)AnalyticsUtils.GetTrackingState() - 1;
             string[] labels = {"Disabled", "Minimum", "Enabled"};
 
-            int newState = EditorGUILayout.Popup("Analytic Tracking", state, labels);
+            int newState = EditorGUILayout.Popup("Analytics Tracking", state, labels);
             if (newState != state)
             {
                 AnalyticsUtils.SetTrackingTo((AnalyticsState)Enum.ToObject(typeof(AnalyticsState), newState + 1));
             }
+
+            GUIStyle hyperlink = new GUIStyle();
+            hyperlink.normal.textColor = new Color(0.122f, 0.435f, 0.949f);
+
+            GUILayout.BeginArea(new Rect(3, 30, 280, 50));
+
+            if (GUILayout.Button("Data Privacy Information", hyperlink, GUILayout.ExpandWidth(false)))
+            {
+                string dataPrivacyStatementUrl = $"https://github.com/Innoactive/Creator/tree/{EditorUtils.GetCoreVersion()}/.Documentation/articles/innoactive-creator/data-privacy-information.md";
```

Comment:
The page won't exist if the Creator is cloned from Github since there is no release version file.
Perhaps, consider opening the local file for those cases.